api/v1beta1/ssp_types.go

```diff
@@ -91,6 +91,22 @@ type SSPSpec struct {
 
 	// CommonInstancetypes is the configuration of the common-instancetypes operand
 	CommonInstancetypes *CommonInstancetypes `json:"commonInstancetypes,omitempty"`
+
+	// TektonPipelines is the configuration of the tekton-pipelines operand
+	TektonPipelines *TektonPipelines `json:"tektonPipelines,omitempty"`
+
+	// FeatureGates is the configuration of the tekton operands
+	FeatureGates FeatureGates `json:"featureGates,omitempty"`
```

Comment:
The feature gate should remain, as we agreed to GA this functionality and remove feature gate in 4.15. Deploying tekton via pointer is not intuitive and can lead to problems. Because even if user does not define the structures it can mean, that user does not want to set anything and would like to use default settings (deploy in the same namespace as CR)